lib/rules/disallow-space-between-arguments.js

```diff
@@ -46,20 +46,13 @@ module.exports.prototype = {
     },
 
     check: function(file, errors) {
-
-        var tokens = file.getTokens();
-
         file.iterateNodesByType(['CallExpression'], function(node) {
-
             node.arguments.forEach(function(param, i) {
-
-                var punctuatorToken = file.getTokenByRangeStart(param.range[1]);
-
+                var punctuatorToken = file.getPrevToken(file.getFirstNodeToken(param));
```

Comment:
why isn't this line just: `var punctuatorToken = file.getFirstNodeToken(param);` previously it was `var punctuatorToken = file.getTokenByRangeStart(param.range[1]);`?